### DISCUSSION TITLE
How to override or replace all key mappings

### DISCUSSION THREAD

**Discussion by ghost** [NvChad/NvChad#1090] [Q&A] [answered]
Hey, 

First of all thanks for the great work with this and making it easy to configure Nvim.

Now to my question, after Reading through the docs for the Custom Mappings, I was wondering how would I override or clear out all default mappings at once. 

Reason I want to do this, since we include Which-Key as a Plugin I wanted to make Use of it specially it's ability to group keybindings and giving them a group name. 

Since currently it only shows `prefix` for me. I want to create My own mappings from the start and use the groups of Which-Key. 

After Checking the code I did not really see a way of doing this besides overwriting or to disable  every key mapping defined in `core.mappings` and over writing the config for the wich-key plugin. 

Am I missing something or is this the only option Atm if it is currently the only Option I could probably write up a quick Pr to include a switch for it.

**Comment by siduck** [comment]
@flammehawk why do yo u want to remove all mappings? you can disable whichkey if you want, your mappings will still be there

**Reply by ghost** [reply]
Thats the Thing I want to use which-key
the Reason would be to be able to set the which-key menu in the way I like it. 
Currently I have multiple Keys that are groups but only show `prefix` as the group name making me question what the contained key mapping actually do.
![Bildschirmfoto 2022-05-16 um 12 17 58](https://user-images.githubusercontent.com/57394581/168571971-6e324d16-2ad7-4fb6-a4b1-7cce2d0f9432.png)

**Comment by LeonHeidelbach** [comment, marked as answer]
@flammehawk @siduck Do it like this. This will NOT override key mappings if you have any set to that specific key:

![Screenshot_20220516_123502](https://user-images.githubusercontent.com/55194243/168574805-ec5c33d7-6952-48e7-b0b6-e921c005e4fe.png)

**Reply by ghost** [reply]
Thanks that is a solution I haven't thought about.

**Reply by Mechanosec** [reply]
does not work for me(((
where I must write that

**Reply by siduck** [reply]
@Mechanosec check recently closed issue regarding whichkey. simple
